kintaro/kintaro.py

```diff
@@ -227,3 +232,20 @@ def run(self, *args, **kwargs):
                 collection_id=kintaro_collection,
                 project_id=self.config.project)
             self.bind_collection(entries, collection_pod_path)
+
+
+def schema_name_to_partial(value, sep='-', directory='partials'):
+    """Parse a kintaro schema name to determine if it is a partial."""
+    if value.lower().startswith('partial'):
```

Comment:
Make this an arg too? I've been calling my schema names "sectionFoo" because that has more meaning than "partial" for non-tech folks. Then you can reuse the arg here and in the next line. :) After that... 👍